.prettierignore

```diff
@@ -0,0 +1,6 @@
+dist/
+coverage/
+node_modules/
+.yarn/
+*.gltf
+*.mdx
```

Comment:
Ignored MDX files, because [semi](https://prettier.io/docs/en/options.html#semicolons) rule set to `false` and it adds semicolons at the beginning of root-level JSX code.

For example, this:

```jsx
import { PerspectiveCamera } from '@react-three/drei'
<Canvas>
  <PerspectiveCamera makeDefault manual />
</Canvas>
```

replaced by this:

```jsx
import { PerspectiveCamera } from '@react-three/drei'
;<Canvas>
  <PerspectiveCamera makeDefault manual />
</Canvas>
```